src/plugin.js

```diff
@@ -666,6 +670,10 @@ const contribAdsPlugin = function(options) {
           if (player.paused()) {
             this.state = 'content-set';
           } else {
+            // If we get here we missed the 'play' event, so pause the player so
+            // that content doesn't start playing immediately after the source change
+            player.pause();
```

Comment:
I think we may want to pause the player if it's not paused yet and then go to the **same** state `content-set`. 

If I understand correctly, the reason this else block will go to `ads-ready?` is that there is an assumption that the `play` event has been missed. So once the player is paused, we should be in the same state that causes the if clause to change state to `content-set`

Reply:
Makes sense to me

Reply:
From an offline conversation we came to the conclusion that the state transition to `ads-ready?` is needed still.